readme.md

```diff
@@ -40,6 +40,10 @@ Once you search you should see a list similar to the one below (versions may be
 | Microsoft.Toolkit.Uwp.Connectivity | API helpers such as BluetoothLEHelper and Networking | 
 | Microsoft.Toolkit.Uwp.DeveloperTools | XAML user controls and services to help developer building their app | 
 
+## <a name="supported"></a> Supported SDKs
+* Anniversary Update (14393)
+* Creators Update (15063)
+
```

Comment:
We should add Fall Creators Update here because the update will be released before v2.1

Reply:
We will add it as soon as it is available